dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/services/HTMLPageAssetRenderedTest.java

```diff
@@ -1259,76 +1277,98 @@ public void shouldReturnParserContainerUUID(final Container container, final Tem
         Assert.assertTrue(html.matches(regexExpected));
     }
 
-    /**
-     * Method to test: {@link com.dotmarketing.portlets.htmlpageasset.business.render.HTMLPageAssetRenderedAPIImpl#getPageHtml(PageContext, HttpServletRequest, HttpServletResponse)}
-     * Given Scenario: Create a page with File Container linked with relative path in the template
-     * ExpectedResult: should work
-     *
-     * @throws Exception
-     */
-    @Test
-    public void shouldRenderRelativeContainerPath() throws Exception {
 
-        final FileAssetContainer container = createFileContainer();
-        final Template template = new TemplateDataGen().title("PageContextBuilderTemplate"+System.currentTimeMillis())
-                .host(site)
-                .withContainer(container.getPath(), UUID)
-                .nextPersisted();
-        PublishFactory.publishAsset(template, systemUser, false, false);
-
-        final String pageName = "testPage-"+System.currentTimeMillis();
-        final HTMLPageAsset pageEnglishVersion = createHtmlPageAsset(template, pageName, 1);
+    @DataProvider(format = "%m page Host: %p[0] Template Host: %p[1] Container Host: %p[2]")
+    public static Object[][] fileContainerCases() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6034685)